File: semestr-8/Jezyk C#/alexander/Csharp/Archiwum/2022s/ID06PD1/Strategy/Strategy/Sportsmen.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Strategy
{
    internal class Sportsmen
    {
        public string surname { get; private set; }
        private SportsmenStrategyRun stRun;
        public Sportsmen(string surname, SportsmenStrategyRun stRun)
        {
            this.surname = surname;
            this.stRun = stRun;
        }
        public string Jump() { return "Jumping"; }
        public string Run() { return stRun.Run(); }
        public string Swim() { return "Swimming"; }
    }
}

File: semestr-8/Programowanie obiektowe w Java/alexander/POJ/Archiwum/2022s/IZ08IO2/Strategy/src/Sportsmen.java
public class Sportsmen
{
   private String surname;
   private SportsmenStrategyRun stRun;
   public Sportsmen(String surname,SportsmenStrategyRun stRun)
   {
      this.surname=surname;
      this.stRun=stRun;
   }
   public String getSurname() { return surname; }
   public String Run() { return stRun.Run(); }
   public String Jump() { return "Jumping"; }
   public String Swim() { return "Swimming"; }
}
